tests/integration/commands/dev/dev-miscellaneous.test.ts

```diff
@@ -237,7 +256,7 @@ describe.concurrent('commands/dev-miscellaneous', () => {
           },
         ])
         .withEdgeFunction({
-          handler: (req, context) =>
+          handler: (_: Request, context: HandlerContext) =>
             Response.json({
               requestID: req.headers.get('x-nf-request-id'),
```

Comment:
Bad copypasta?

```suggestion
          handler: (req: Request, context: HandlerContext) =>
            Response.json({
              requestID: req.headers.get('x-nf-request-id'),
```

Reply:
fyi, you're reviewing a `wip` commit here.